types/dataframe-js/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for dataframe-js 1.0
+// Project: https://github.com/Gmousse/dataframe-js#readme
+// Definitions by: Alzemiro Iago de Moraes Thomaz <https://github.com/alzemiro-iago>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
+
+import DataFrame from './dataframe';
+import Row from './row';
+import * as Errors from './errors';
+import * as Reusables from './reusables';
+import * as Io from './io';
+import * as Symbol from './symbol';
+import { GroupedDataFrame } from './group';
+import SQL from './modules/sql';
+import Matrix from './modules/matrix';
+import Stat from './modules/stat';
+import sqlParser from './modules/sql/sqlEngine';
+export as namespace DataFrame;
+export { Row, Errors, SQL, DataFrame, GroupedDataFrame, Io, Reusables, Symbol, sqlParser, Matrix, Stat };
```

Comment:
```suggestion
export { Row, Errors, SQL, DataFrame, GroupedDataFrame, Io, Reusables, Symbol, sqlParser, Matrix, Stat };
export default DataFrame;
```

Reply:
OK, I've missed details in the module, as it adds symbols to exported modules, like with `DataFrame`.
I understand this was the pattern you're trying to use to add additonal symbols, like `DataFrame.sql`
you can also evaluate below pattern, instead of using namespaces:
```ts
import DataFrame from './dataframe'
....
declare const _default: typeof DataFrame & {
    sql: typeof SQL;
};

export { Row, Errors, _default as DataFrame };
export default _default;
````
later:
```ts
DataFrame.setDefaultModules();
DataFrame.sql.request('SELECT * FROM tmp2 WHERE column1 = 6')
```

Reply:
Okay, I'll validate this suggested pattern. I'll run some tests to make sure it doesn't break anything in the original code. I will need 2 days for this.

Reply:
Good evening, as I am not the owner of the project and as long as all tests are ok, can we approve this PR? I mean, in the model that is today, without any further modification.